npm/esbuild-wasm/bin/esbuild

```diff
@@ -44,9 +45,14 @@ process.on('exit', code => {
   const data = require('../exit0')[nativeModule];
   if (data) {
     try {
-      const tempFile = path.join(os.tmpdir(), `${Math.random().toString(36).slice(2)}-${nativeModule}`);
-      fs.writeFileSync(tempFile, zlib.inflateRawSync(Buffer.from(data, 'base64')));
-      require(tempFile);
+      const hash = crypto.createHash('sha256').update(data).digest().toString('hex').slice(0, 16);
+      const tempFile = path.join(os.tmpdir(), `${hash}-${nativeModule}`);
+      try {
+        require(tempFile);
```

Comment:
I think the contents of the file should probably be validated here before using it.

Reply:
I had thought about that. It would be yet another read, but sure.

Reply:
done